.github/workflows/ci.yml

```diff
@@ -41,26 +41,21 @@ jobs:
           - 9200:9200
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
       - name: update apt
         run: sudo apt-get update -y
 
       - name: Apt install
         run: cat Aptfile | sudo xargs apt-get install
 
-      - name: Set up Python
-        uses: actions/setup-python@v2
+      - uses: actions/setup-python@v4
         with:
-          python-version: "3.9.13"
-
-      - id: cache
-        uses: actions/cache@v1
```

Comment:
We no longer need to use `actions/cache` directly. Both `actions/setup-python` and `actions/setup-node` support caching of pip and yarn dependencies.

In the case of NodeJS dependencies, caching means "no need to download the package". It still needs to be copied from the package manager's global cache into `node_modules`.